<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMbox</class>
 <widget class="QWidget" name="QMbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMbox</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/eventq/png/弹窗提示/弹窗-底.png);</string>
   </property>
   <widget class="QLabel" name="label_content">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/eventq/png/全透明.png);color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-image: url(:/eventq/png/全透明.png);</string>
    </property>
    <property name="text">
     <string>提示</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>226</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/eventq/png/全透明.png);</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_OK">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/eventq/png/弹窗提示/弹窗-按钮1.png);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>确认</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_NO">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/eventq/png/弹窗提示/弹窗-按钮2.png);</string>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="eventplclent.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_NO</sender>
   <signal>clicked()</signal>
   <receiver>QMbox</receiver>
   <slot>slots_NO()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>QMbox</receiver>
   <slot>slots_OK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slots_OK()</slot>
  <slot>slots_NO()</slot>
 </slots>
</ui>
